src/litellm/proxy_config.yaml

```diff
@@ -10,3 +10,11 @@ model_list:
       model: ollama/phi
 litellm_params:
   drop_params: True
+
+litellm_params:
+  drop_params: True
+
+router_settings:
+  routing_strategy: least-busy      # Literal ["simple-shuffle", "least-busy", "usage-based-routing", "latency-based-routing"]
```

Comment:
Remove comments. Comment should be used to explain the codes. For everything else there is git.